perun-base/src/main/java/cz/metacentrum/perun/core/api/exceptions/ConsistencyErrorException.java

```diff
@@ -2,21 +2,34 @@
 
 
 /**
- * This exception raises when some inconsistency in underlaying data sources occur.
+ * This exception raises when some inconsistency in underlying data sources occurs.
```

Comment:
I would add something like "Usually, it cannot be solved on the client side, but it have to be fixed by Perun developers."